platform/security/handler/security-handler-pki/src/test/java/org/codice/ddf/security/handler/pki/CrlCheckerTest.java

```diff
@@ -118,25 +124,42 @@ public void testGetPropertiesFailsBothCertsPass() throws CertificateException {
         certs = extractX509CertsFromString(certificateString);
         assertThat(crlChecker.passesCrlCheck(certs), equalTo(true));
     }
+    
+    @Test
+    public void testIsCrlDisabled() {
+        CrlChecker crlChecker = getConfiguredCrlChecker("encryption-crl-none.properties");
+        
+        boolean enabled = crlChecker.isCrlEnabled();
+        
+        assertThat(enabled, equalTo(false));
+    }
+    
+    @Test
+    public void testIsCrlEnabled() {
+        CrlChecker crlChecker = getConfiguredCrlChecker("encryption-crl-valid.properties");
+        
+        boolean enabled = crlChecker.isCrlEnabled();
+        
+        assertThat(enabled, equalTo(true));
+    }
 
     private CrlChecker getConfiguredCrlChecker(String encryptionProperties) {
-        CrlChecker crlChecker = new CrlChecker();
-        Properties prop = crlChecker.loadProperties(encryptionProperties);
-        String crlPropertyValue = prop.getProperty(crlChecker.CRL_PROPERTY_KEY);
-
-        // Prevents a null pointer in the unit tests when appending the unit test's getResource path
-        if (crlPropertyValue == null) {
-            crlChecker.setCrlLocation(crlPropertyValue);
-        } else {
-            String crlRelativePath = "/" + prop.getProperty(crlChecker.CRL_PROPERTY_KEY);
-            String crlAbsolutePath = PKIHandlerTest.class.getResource(crlRelativePath)
-                    .getPath();
-            crlChecker.setCrlLocation(crlAbsolutePath);
-        }
-
+        CrlChecker crlChecker = new CrlChecker() {
+            @Override
+            Path getCrlPathFromPropertiesFile() {
```

Comment:
Isn't there a way to only override `loadProperties` instead of having to override the whole `getCOnfigurationCrlChecker()` method? This probably kills our code coverage.